src/install.rs

```diff
@@ -1558,6 +1560,19 @@ fn pre_build_command(config: &Config, dir: &Path, base: &str, version: &str) ->
     Ok(())
 }
 
+fn post_build_command(config: &Config, dir: &Path, base: &str, version: &str) -> Result<()> {
```

Comment:
This is basically identical to the function above. Maybe we turn this into just build_command() where we pass in the command then have tiny functions for pre and post build.